src/ui/pages/WalletSelect/WalletSelect.tsx

```diff
@@ -108,7 +108,10 @@ export function WalletSelect() {
   );
 
   const { data: walletsMeta, isLoading: isLoadingWalletsMeta } =
-    useWalletsMetaByChunks({ addresses: ownedAddresses });
+    useWalletsMetaByChunks({
```

Comment:
wallet meta request that blocks render make wallet list opens very slow if fails, but probably this is out of scope for this fix

Reply:
Yep, I think we can add a smaller timeout here, but I also think it can be done separately